webApps/apps.json

```diff
@@ -160,5 +160,61 @@
         }
       ]
     }
+    ,
+    {
+      "id": "com.github.owncloud.web-app-dicom-viewer",
+      "name": "DICOM Viewer",
+      "subtitle": "Preview DICOM files (.dcm) – both the medical images and their corresponding metadata.",
```

Comment:
Could you add a `description` please? There are several CSP rules that need to be added to make the app work. That would be important to have in the description:

```
  worker-src:
    - '''self'''
    - 'blob:'
  child-src:
    - 'blob:'
  connect-src:
    - 'data:'
    - 'https://unpkg.com/detect-gpu@5.0.38/dist/benchmarks/d-apple.json'
  script-src:
    - '''unsafe-eval'''
```